frontend/src/utils/helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { HapiTelecomItem, HapiTelecomType } from '../types/Hapi.types';
+
+export const getPractitionerEmail = (telecomItems: HapiTelecomItem[]) => {
+  let email: string = '';
+
+  if (telecomItems) {
+    const foundEmail: HapiTelecomItem[] = telecomItems.filter((ti) => ti.system === HapiTelecomType.EMAIL);
```

Comment:
The Hapi API is not great in terms of the data schema. Having to do things like this is annoying.

Reply:
Yeah I was going to say this on a follow up call -- one of the most jank apis I've worked with and, if I came across this during some research task or during my day job, I'd work with the backend to sanitize results and shape the data into a more reasonable dto for the frontend (i.e. fetch to backend, reorganize, expose endpoint to FE). There's a lot of unnecessary values the frontend doesn't need to know about and just adds complexity. I think the frontend should only be about displaying data -- not having to process anything (or at least minimal processing of data).